lib/auth/oidc.go

```diff
@@ -810,21 +809,21 @@ func (a *Server) getClaims(oidcClient *oidc.Client, connector services.OIDCConne
 	var exists bool
 	if idsub, exists, err = idTokenClaims.StringClaim("sub"); err != nil || !exists {
 		log.Debugf("Unable to extract OIDC sub claim from ID token.")
-		return nil, trace.Wrap(err)
+		return nil, trace.Wrap(err, "unable to extract OIDC sub claim from ID token")
 	}
 	if uisub, exists, err = userInfoClaims.StringClaim("sub"); err != nil || !exists {
 		log.Debugf("Unable to extract OIDC sub claim from UserInfo.")
-		return nil, trace.Wrap(err)
+		return nil, trace.Wrap(err, "unable to extract OIDC sub claim from UserInfo")
 	}
 	if idsub != uisub {
-		log.Debugf("OIDC claim subjects don't match '%v' != '%v'.", idsub, uisub)
-		return nil, trace.BadParameter("invalid subject in UserInfo")
+		log.Debugf("OIDC claim subjects does not match '%v' != '%v'.", idsub, uisub)
```

Comment:
I think it was grammatically correct before?

Reply:
reverted